docs/codingpractises.md

```diff
@@ -0,0 +1,19 @@
+# Coding Practises
+
+The [contributing.md](Contributing Guide) provides (among other things) guidance on the workflows and general expectations associated with contributing a code change to `scores`. This document eschews some of the context to focus on specifying technical information needed when developing code for `scores`.
+
+# Expectations:
```

Comment:
Can we also have `isort --profile=black` has been run over the code?

Reply:
@nicholasloveday this  has been a default setting in `pyproject.toml` since migrating to github.
https://github.com/nci/scores/blame/develop/pyproject.toml#L70

Reply:
Bit of a nit pick but can we try to use snake-case naming conventions for markdown file names?
e.g `docs/coding_practices`

Also not a nit, the name is a typo
 ```diff
- docs/codingpractises.md
+ docs/coding_practices.md
```

Reply:
No problem to rename it to coding_practises if that's more seemly. Feel free to add that to the MR yourself, otherwise I'll get to it when I get a moment.

Reply:
A couple pushes later and I've think I've got a final version here.
https://github.com/nci/scores/blob/36-add-type-hints-for-public-functions/docs/coding_practices.md